pkgs/os-specific/linux/zfs/generic.nix

```diff
@@ -46,7 +46,9 @@ let
       extraLongDescription ? "",
       extraPatches ? [ ],
       rev ? "zfs-${version}",
-      kernelCompatible ? null,
+      kernelMinSupportedMajorMinor,
+      kernelMaxSupportedMajorMinor,
+      kernelMaxBuildableMajorMinor ? kernelMaxSupportedMajorMinor, # Only use for pre-release ZFS
```

Comment:
Could we make this a boolean instead? something like `ignoreKernelCompatibility`